webassembly-observe-sdk/README.md

```diff
@@ -0,0 +1,53 @@
+# Dylibso WebAssembly Observe SDK
+
+## Overview
```

Comment:
The overview should include: 

1. What does WebAssemble do in 1-2 sentences?
2. What will customers get from this integration?
3. What types of data (and the value of this data)?